tests/system/src/testscriptapp1/java/mymod/myapp1/FXMLScriptTest.java

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package myapp1;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeMap;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.input.MouseButton;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+import javax.script.Bindings;
+import javax.script.ScriptContext;
+
+import static myapp1.Constants.*;
+import pseudoScriptEngine.InvocationInfos;
+import pseudoScriptEngine.RgfPseudoScriptEngine;
+
+/**
+ * Modular test application for testing FXML.
+ * This is launched by ModuleLauncherTest. To only run the ModuleLauncherTest:
+ * <code>./gradlew -PFULL_TEST=true :systemTests:test --tests ModuleLauncherTest</code>
+ */
+public class FXMLScriptTest extends Application {
+
+    static boolean bDebug = false; // true; // display invocation list
+
+    /** Runs the application and invokes the tests.
+     *  @param args the command line arguments, if any given the RgfPseudoScriptEngine invocation logs get displayed
+     *              which are used in the asserCorrectInvocations() method
+     */
+    public static void main (String args[]) {
+        try {
+            // any argument will cause the bDebug flag to be set to true
+            if (args.length>0) {
```

Comment:
We usually add a space before and after a binary operator (this one has exceptions, but in this case it would be better with the extra spaces).